rust_src/src/casefiddle.rs

```diff
@@ -215,32 +215,16 @@ pub extern "C" fn syms_of_casefiddle() {
 #[no_mangle]
 pub extern "C" fn keys_of_casefiddle() {
     unsafe {
-        initial_define_key(
-            control_x_map,
-            Ctl('L'),
-            CString::new("downcase-region").unwrap().as_ptr(),
-        );
-        initial_define_key(
-            control_x_map,
-            Ctl('U'),
-            CString::new("upcase-region").unwrap().as_ptr(),
-        );
-
-        initial_define_key(
-            meta_map,
-            'c' as i32,
-            CString::new("capitalize-word").unwrap().as_ptr(),
-        );
-        initial_define_key(
-            meta_map,
-            'l' as i32,
-            CString::new("downcase-word").unwrap().as_ptr(),
-        );
-        initial_define_key(
-            meta_map,
-            'u' as i32,
-            CString::new("upcase-word").unwrap().as_ptr(),
-        );
```

Comment:
Maybe we could add a macro for these strings?

Reply:
I was thinking the same thing.